easybuild/easyblocks/p/psi.py

```diff
@@ -150,11 +156,30 @@ def configure_step(self):
             if self.name == 'PSI4':
                 pcmsolverroot = get_software_root('PCMSolver')
                 if pcmsolverroot:
-                    self.cfg.update('configopts', " -DENABLE_PCMSOLVER=ON -DPCMSOLVER_ROOT=%s" % pcmsolverroot)
+                    self.cfg.update('configopts', " -DENABLE_PCMSOLVER=ON")
+                    if LooseVersion(self.version) < LooseVersion("1.2"):
+                        self.cfg.update('configopts', " -DPCMSOLVER_ROOT=%s" % pcmsolverroot)
+                    else:
+                        self.cfg.update('configopts', " -DCMAKE_INSIST_FIND_PACKAGE_PCMSolver=ON "
+                                        "-DPCMSolver_DIR=%s/share/cmake/PCMSolver" % pcmsolverroot)
 
                 chempsroot = get_software_root('CheMPS2')
                 if chempsroot:
-                    self.cfg.update('configopts', " -DENABLE_CHEMPS2=ON -DCHEMPS2_ROOT=%s" % chempsroot)
+                    self.cfg.update('configopts', " -DENABLE_CHEMPS2=ON")
+                    if LooseVersion(self.version) < LooseVersion("1.2"):
+                        self.cfg.update('configopts', " -DCHEMPS2_ROOT=%s" % chempsroot)
+                    else:
+                        self.cfg.update('configopts', " -DCMAKE_INSIST_FIND_PACKAGE_CheMPS2=ON "
+                                        "-DCheMPS2_DIR=%s/share/cmake/CheMPS2" % chempsroot)
+
+                #  Be aware, PSI4 wants exact versions of the following deps! built with CMake!!
```

Comment:
What happens if you use the wrong version? How can one figure out which versions PSI4 wants?

Reply:
If we use the wrong version then PSI4 will not find them. Otherwise you have to patch the corresponding `CMakeLists.txt` files in the source.

Reply:
@hajgato Please mention the latter in the comment in the easyblock?